plugins/aeroscope.py

```diff
@@ -26,6 +30,12 @@
 )
 
 
+class AeroForm(Form):
+    company = StringField("Company", [validators.Length(min=4, max=25)])
+    email = StringField("Email Address", [validators.Email()])
```

Comment:
Why add email?

Reply:
There is a lookup we can do to automatically connect it via salesforce. Company was a fallback, if they weren't a user in Salesforce

Reply:
Let's leave it with just `organization` for now